modules/statistics/AvgRTPStatsReporter.js

```diff
@@ -223,26 +219,33 @@ class ConnectionAvgStats {
             return;
         }
 
+        if (!data.transport || !data.transport.length) {
```

Comment:
not only that, but it's not clear why the transport needs to be an array.. In my experiments I haven't observed any cases where the transport is not set to something.